### PR TITLE
feat(docs/SRS): Add Section 2: Stakeholders

### PR DESCRIPTION
**Pull request by MitchHynes**
merge stakeholders sections into main branch.

### CODE REVIEW

File: docs/SRS-Volere/SRS.tex [hunk 1]
```diff
@@ -60,21 +60,131 @@ \subsection{Goals of the Project}
...
+        on the application, he receives a notification that a contractor he manages authenticated at site
+        to complete a work order, but the application identified they had not completed their
+        health and safety training. Greg prompts the system to issue this training to them, and runs a report
+        on this contractor to see if any other employees of theirs require their training soon.
+    \end{itemize}
+    \item Nancy: A newly onboarded contractor
+    \begin{itemize}
+        \item[-] Nancy is a plumber who is servicing her first work order after being hired by "Worldly Plumbing".
+        Her company has notified her of the application she must use to authenticate for the work.
+        When she authenticates, she is notified that the device she is servicing is located within a confined space
```
Comment (kyle4908) [inline docs/SRS-Volere/SRS.tex:156]: I think this and a couple others go over the 80 char line length limit https://github.com/Spitgranger/capstone/blob/main/docs/Checklists/Writing-Checklist.pdf

Comment (MitchHynes) [reply docs/SRS-Volere/SRS.tex:156]: thanks,

just found a way to have vim have a char limit indicator, use:
set colorcolumn=80

image

It looks like some of the lines provided by the template itself are over the character limit

Comment (MitchHynes) [reply docs/SRS-Volere/SRS.tex]: fixed.

File: docs/SRS-Volere/SRS.tex [hunk 2]
```diff
@@ -60,21 +60,154 @@ \subsection{Goals of the Project}
...
+        he manages authenticated at site to complete a work order, but the 
+        application identified they had not completed their health and safety 
+        training. Greg prompts the system to issue this training to them, and 
+        runs a report on this contractor to see if any other employees of 
+        theirs require their training soon.
+    \end{itemize}
+    \item Nancy: A newly onboarded contractor
+    \begin{itemize}
+        \item[-] Nancy is a plumber who is servicing her first work order after 
+        being hired by "Worldly Plumbing".
```
Comment (kyle4908) [inline docs/SRS-Volere/SRS.tex:170]: I see this highlighted as a syntax error. I see in all of Dr. Smith's files he uses `` and '' to indicate the start and end of quotes, so maybe you need to do that (i.e. [here](https://github.com/Spitgranger/capstone/blob/39383f8670ec917d4a2878079708853a4f3c74dd/docs/Checklists/Writing-Checklist.tex#L111)). Like:
```suggestion
        being hired by ``Worldly Plumbing''.
```

Comment (MitchHynes) [reply docs/SRS-Volere/SRS.tex]: Thanks, fixed the quotes.

**Review by kyle4908 — APPROVED** [review]

**Review by Spitgranger — APPROVED** [review]

**Pull request merged**